modules/packages/msiexec.bat

```diff
@@ -96,8 +96,12 @@ rem Install this file if it exists
     goto :EOF
   )
 
-  %MSIEXEC% /quiet /passive /qn /norestart /i %1
-  rem TODO options, error checking
+  REM TODO: ENT-6824 save this logfile based on msi filename
+  set logfile="\cfengine_package_install.log"
+  %MSIEXEC% /quiet /passive /qn /norestart /l*vx %logfile% /i %1

```

Comment:
I added both v and x options to the `/l` argument as they seemed "different" in the help/usage given by msiexec.exe dialog popup. In practice I didn't see much difference but figured "might as well".